gatsby-node.js

```diff
@@ -36,9 +36,10 @@ exports.onCreateNode = ({node, boundActionCreators, getNode}) => {
   let slug;
   switch (node.internal.type) {
     case `MarkdownRemark`:
+      const fmSlug = (node.frontmatter && node.frontmatter.slug) ? node.frontmatter.slug : false
```

Comment:
Here you can use `get` from lodash (https://lodash.com/docs/4.17.5#get), and I prefer long variable name instead of acronyms :wink:

```js
const frontmatterSlug = get(node, 'frontmatter.slug');
```